<launch> 
    <arg name="vehicle_name" default="bluerov" />
    <group ns="$(arg vehicle_name)">
        <node name="trajectory_generator" pkg="bluerov_control" type="trajectory_generator.py"></node>
    </group>
</launch>